### ISSUE TITLE
Add multiple annotations

### ISSUE THREAD

**Issue by felixalguzman** [apexcharts/apexcharts.js#36]
Is there a way to add multiple annotations based on array of data ?

something like this
![imagen](https://user-images.githubusercontent.com/17243615/44166575-b2806700-a099-11e8-9623-0e7115793775.png)

**Comment by junedchhipa** [comment]
Are you looking for something like this?
![screen shot 2018-08-16 at 12 17 15 am](https://user-images.githubusercontent.com/17950663/44166760-d8cdf300-a0e9-11e8-9ee7-764634b2b2f8.png)

The dataLabels can also be formatted. Read more here
https://apexcharts.com/docs/options/datalabels/

**Comment by felixalguzman** [comment]
Yes but the values are not the same in the Y axis, I want to put an array of calculated data, I am not sure if this is possible.

**Comment by junedchhipa** [comment]
Here is a [codepen](https://codepen.io/apexcharts/pen/yxePzM) example
